hack/tools/configure_vault_kind_multi.sh

```diff
@@ -55,19 +67,9 @@ case "$op" in
         configure_vault
```

Comment:
Is this required for storing Egeria credentials? Shouldn't $SECRET_PATH point to /external instead of /m4d/dataset-creds ?

Reply:
configure_vault adds a policy to access $PLUGIN_PATH and $SECRET_PATH. The later is for Egeria (should be removed soon). External is where the user insert the policy but vault connector copies it to $SECRET_PATH so $SECRET_PATH is the one that is mentioned in the policy as the modules eventually access this path.